Code/Framework/AzCore/AzCore/Memory/OSAllocator.h

```diff
@@ -62,13 +61,12 @@ namespace AZ
         size_type       Capacity() const override                { return m_custom ? m_custom->Capacity() : AZ_CORE_MAX_ALLOCATOR_SIZE; } // custom size or unlimited
         size_type       GetMaxAllocationSize() const override    { return m_custom ? m_custom->GetMaxAllocationSize() : AZ_CORE_MAX_ALLOCATOR_SIZE; } // custom size or unlimited
         size_type       GetMaxContiguousAllocationSize() const override { return m_custom ? m_custom->GetMaxContiguousAllocationSize() : AZ_CORE_MAX_ALLOCATOR_SIZE; } // custom size or unlimited
-        IAllocatorAllocate*  GetSubAllocator() override          { return m_custom ? m_custom : NULL; }
          
     protected:
         OSAllocator(const OSAllocator&);
         OSAllocator& operator=(const OSAllocator&);
 
-        IAllocatorAllocate*     m_custom;
+        IAllocatorSchema*     m_custom;
```

Comment:
Is this member needed anymore on `OSAllocator` as AllocatorBase inherits from `IAllocator` which as an `IAllocatorSchema` member already.

Reply:
still needed, in some restricted platform, it uses this m_custom entry to do allocations. This is not the proper way to PALify it, but will leave that refactoring for a future pass.